### PR TITLE
systemd-notify: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-notify.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Notify systemd that the service has completed its initialization and is fully started. It should be invoked when the service is ready to accept incoming requests:
+
+`systemd-notify --booted`
+
+- Signal to systemd that the service is ready to handle incoming connections or perform its tasks:
+
+`systemd-notify --ready`
+
+- Provide a custom status message to systemd. It can be used to provide additional information about the service's initialization progress or state:
```
Comment (acuteenvy) [inline pages/linux/systemd-notify.md:14]:
```suggestion
- Provide a custom status message to systemd (this information is shown by `systemctl status`):
```

File: pages/linux/systemd-notify.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`systemd-notify --status="Add custom status message here..."`
```
Comment (acuteenvy) [inline pages/linux/systemd-notify.md:16]:
```suggestion
`systemd-notify --status="{{Add custom status message here...}}"`
```

File: pages/linux/systemd-notify.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# systemd-notify
+
+> Notify service manager about start-up completion and other daemon status changes.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-notify.html#>.
```
Comment (acuteenvy) [inline pages/linux/systemd-notify.md:4]:
```suggestion
> Notify the service manager about start-up completion and other daemon status changes.
> This command is useless outside systemd service scripts.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-notify.html>.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by quantumflo** [comment]
> LGTM, Thanks for your contribution.

Will contribute more soon 👍

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**